estate/models/estate.py

```diff
@@ -0,0 +1,84 @@
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+class PartnerModel(models.Model):
+    _inherit = "res.partner"
+    property_ids = fields.One2many('estate.property', 'buyer')
+
+class UserModel(models.Model):
+    _inherit = "res.users"
+    property_ids = fields.One2many('estate.property', 'seller')
+
+
+class EstateModel(models.Model):
+    _name = "estate.property"
+    _description = "Real estate properties"
```

Comment:
In python, we use single quote around technical terms like `_name` here, but we use double quotes around "human" text like you did for `_description`. (E.g.the `string` fields in field definitions, the label of selection values...)